src/constants/blog.ts

```diff
@@ -0,0 +1 @@
+exports.postsPerPage = 10
```

Comment:
```suggestion
exports.postsPerPage = 10

```

Reply:
Also can't this be `export const postsPerPage = 10`?

Reply:
I was getting an error with that, since we're importing it to `gatsby-node.js`. Not sure if it's related to webpack or the version of Node I have installed locally. I'll try updating node and changing this

Reply:
Eek yeah that makes sense. I guess we have to live with that. Annoying that gatsby-node doesn't have TS support. Maybe just put a comment at the top of the file explaining why we are using that syntax (so that it can be used both at build and runtime) and then we should be gtg.